modules/apps/portal-search-elasticsearch7/portal-search-elasticsearch7-impl/src/main/java/com/liferay/portal/search/elasticsearch7/internal/information/ElasticsearchSearchEngineInformation.java

```diff
@@ -190,6 +190,10 @@ private void _addActiveConnections(
 			filterString);
```

Comment:
Where ever connectionInformationList is being passed from, we should have logging on that.